Week4/ReadingList/app/src/main/java/com/florencenjeri/readinglist/LogInFragment.kt

```diff
@@ -0,0 +1,70 @@
+package com.florencenjeri.readinglist
+
+import android.os.Bundle
+import android.text.TextUtils
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.florencenjeri.readinglist.model.UserPrefs
+import kotlinx.android.synthetic.main.fragment_log_in.*
+
+class LogInFragment : Fragment() {
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        setHasOptionsMenu(false)
+        //Check if the user is logged in
+        if (UserPrefs.isUserLoggedIn()) {
```

Comment:
Don't do this logic in `onCreateView`, it should only be used for View building. Do this in onViewCreated!